extending/template_system/search_table.rst

```diff
@@ -0,0 +1,125 @@
+.. _template_search_table:
+
+
+HTML output for :command:`conan search` table
+=============================================
+
+.. warning::
+
+    This has to be an considered as an **experimental** feature, we might
+    change the context provided to this templates once we have more exmpales
+    from the community.
+
+
+The :ref:`conan_search` command can generate an HTML table with the results of the
+query when looking for binaries
+
+.. image:: ../../images/conan-search_binary_table.png
+    :height: 500 px
+    :width: 600 px
+    :align: center
+
+
+This is the default Canan provides, but you can use your own Jinja2 template to customize
+this output to your needs:
+
+ * *<cache>/templates/output/search_table.html*.
+ 
+
+Context
+-------
+
+Conan feeds this template with the information about the packages found, this information
+is called context and it contains these objects:
+
+ * ``base_template_path``: path to the directory where the choosen template file is
+   located. It is needed if your output file needs to link assets distributed together
+   with the template file.
+ * ``search``: it contains the pattern used in the command line to search packages.
+ * ``results``: this object contains all the information retrieved from the remotes, it
+   is used to get the headers and the rows.
+
+
+When the output is a table, the first thing needed are the headers, these can be a sinle row
+or two rows like the image above. In order to get the headers you should use
+``results.get_headers(keys)`` with a list of extra ``keys`` you want to include. Conan will
```

Comment:
it's not obvious here where should I use `results.get_headers`, in conanfile or somewhere else? I guess link to the reference would be nice here

Reply:
This file is the reference for these classes/methods/variables. I'll add "see example below" here, it can help with user expectations.